index.test.js

```diff
@@ -66,6 +66,34 @@ describe('Remedian algorithm implementation correctness', () => {
     expect(remedian.bufferSize).toEqual(3);
   });
 
+  test('buffer size <= 1, throw an error', () => {
+    expect(()=>{
+      const remedian = new Remedian(-1);
+    }).toThrow("Remedian buffer size must be an odd number greater or equal to 3")
+
+    expect(()=>{
+      const remedian = new Remedian(0);
+    }).toThrow("Remedian buffer size must be an odd number greater or equal to 3")
```

Comment:
*suggestion:* here you have a lot of repetition, plus two test cases are unnecessary. You can look at line 13 of a test file to see how to run the same test with a different argument. Do the same for the bufferSize test case.